Dockerfile

```diff
@@ -10,4 +10,5 @@ RUN tar xzvf apache-maven-3.8.6-bin.tar.gz -C/opt/
 ENV PATH="${PATH}:/opt/apache-maven-3.8.6/bin"
 
 RUN apt-get -y install sudo
-VOLUME /submissions
+
+VOLUME ["/root/.m2","/submissions"]
```

Comment:
we don't want to hardcode source paths since we want to set those at runtime in the .config file